src/elements/content-uploader/__tests__/UploadState.test.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { render, screen } from '../../../test-utils/testing-library';
+
+import UploadState, { UploadStateProps } from '../UploadState';
+
+import { VIEW_ERROR, VIEW_UPLOAD_EMPTY, VIEW_UPLOAD_IN_PROGRESS, VIEW_UPLOAD_SUCCESS } from '../../../constants';
+
+describe('elements/content-uploader/UploadState', () => {
+    const getWrapper = (props: Partial<UploadStateProps>) =>
+        render(
+            <UploadState
+                canDrop={false}
+                hasItems={false}
+                isFolderUploadEnabled={false}
+                isOver={false}
+                isTouch={false}
+                onSelect={jest.fn()}
+                view={VIEW_ERROR}
+                {...props}
+            />,
+        );
+
+    test.each`
+        view                       | iconClassName
+        ${VIEW_ERROR}              | ${'upload-error-state'}
+        ${VIEW_UPLOAD_EMPTY}       | ${'upload-empty-state'}
+        ${VIEW_UPLOAD_IN_PROGRESS} | ${'upload-empty-state'}
+        ${VIEW_UPLOAD_SUCCESS}     | ${'upload-success-state'}
```

Comment:
we should give aria labels to the svgs to query instead of using class names

Reply:
I was trying to find a prop that could help me test those four states together which is className in this case. But I am happy to add aria-label to all those icons. few questions tho:
1.  are aria-labels always necessary for svg components?
2. why is aria-label better to be used for jest testing comparing to classNames, or is there any other prop we should use instead like 'data-testid'